contracts/ERC721M.sol

```diff
@@ -12,6 +12,9 @@ import "./IERC721M.sol";
 contract ERC721M is IERC721M, ERC721AQueryable, Ownable {
     using ECDSA for bytes32;
 
+    address private constant CROSSMINT_ADDRESS =
```

Comment:
Just thinking, in case of some emergency like crossmint private key is compromised, do we want to be able to update this?

Reply:
yeah, I think make it configurable also helps with testing.

Reply:
Done. Made it configurable like cosigner.